docs/user-guide/data-structures.ipynb

```diff
@@ -345,15 +356,51 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d.meta['aux']"
+    "a.meta['aux']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unlike `values` when creating a variable, `data` as well as entries in the meta data dicts (`coords`, `masks`, and `attrs`) are *not* deep-copied on insertion into a data array.\n",
+    "To avoid anwanted sharing, call the `copy()` method.\n",
+    "Compare:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x2 = sc.zeros(dims=['x'], shape=[3])\n",
+    "a.coords['x2_shared'] = x2\n",
+    "a.coords['x2_copied'] = x2.copy()\n",
+    "x2 += 123\n",
+    "a"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Meta data can be removed in the same way as in Python dicts:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "del a.attrs['aux']"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Further details about data arrays are implicitly discussed in the next section, which is covering datasets, since each item in a dataset behaves equivalently to a data array.\n",
-    "\n",
     "### Distinction between dimension coords and non-dimension coords, and aligned and unaligned coords\n",
```

Comment:
In this section, the title talks about `unaligned coords` which is a concept that doesn't really appear anywhere else (i think). Isn't it a remnant of the old naming of `attrs`?
Maybe we should rewite this a bit clearer, especially the last two sentences in the section talking about coord "alignment"? Maybe saying that coords are compared in ops and attrs are not?